index.html

```diff
@@ -303,7 +303,7 @@ <h3>Normative Specifications</h3>
                     <dt id="hash" class="spec">RDF Dataset Hash (RDH)</dt>
                     <dd>
                         <p>
-                            This specification details the processing steps of a hash function for an arbitrary RDF Dataset. These step include (1) the generation of a <a href="./explainer.html#canonical_form">canonical form</a> using the algorithm specified in the “RDF Dataset Canonicalization” deliverable, (2) sorting the <a href="https://www.w3.org/TR/n-quads/">N-Quads</a> serialization of the canonical form generated in the previous step, and (3) application of a cryptographic hash function on the sorted <a href="https://www.w3.org/TR/n-quads/">N-Quads</a> representation. 
+                            This specification details the processing steps of a hash function for an arbitrary RDF Dataset. These step include (1) the generation of a <a href="./explainer.html#canonical_form">canonical form</a> for the RDF Dataset using the algorithm specified in the “RDF Dataset Canonicalization” deliverable, (2) application of a hash function over the canonical form of the RDF Dataset. 
```

Comment:
Because including only one example format, and only including that when *sorted*, suggests that "sorted N-Quads" is preferred or even necessary.
```suggestion
                            This specification details the processing steps of a hash function for an arbitrary RDF Dataset. These step include (1) the generation of a <a href="./explainer.html#canonical_form">canonical form</a> for the RDF Dataset using the algorithm specified in the “RDF Dataset Canonicalization” deliverable, (2) application of a hash function over a serialization (such as sorted <a href="https://www.w3.org/TR/n-quads/">N-Quads</a>, unsorted <a href="https://www.w3.org/TR/n-quads/">N-Quads</a>, or <a href"https://www.w3.org/TR/trig/">TriG</a>) of the canonical form of the RDF Dataset.
```